packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx

```diff
@@ -59,6 +58,9 @@ const useStyles = makeStyles(({ palette }) => ({
   },
   profilePictureImage: {
     width: 'auto'
+  },
+  textFields: {
+    paddingTop: spacing(12)
```

Comment:
padding changes from 64 to 48. making sure that this was on purpose

Reply:
I can count clearly 😫